src/demo/helpers.js

```diff
@@ -66,3 +67,23 @@ export const randomInt = (min, max) => {
 export const clamp = (value, min, max) => {
   return Math.min(Math.max(value, min), max);
 };
+
+// Given a key, returns the value from the browser's current query params.
+export const queryStrFor = key => {
+  return queryString.parse(location.search)[key];
+};
+
+export const filterFishComponents = (fishComponents, dataSet) => {
```

Comment:
A comment on what this function does, and maybe a sample input & output, would be quite helpful.